doc/workflows/terraform_apply.yml

```diff
@@ -0,0 +1,18 @@
+name: Terraform Apply
+
+on:
+  workflow_dispatch: null
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  tacos-gha:
+    uses: getsentry/tacos-gha/.github/workflows/tacos_apply.yml@stable
+    with:
+      tacos_gha_ref: refs/heads/stable
+# Add keys here if your terraform/terragrunt gets modules from a private repo.
```

Comment:
Should probably clarify this a little: keys would be necessary if you're pulling from *another* private repo (not the one where the workflow is running, GHA already has a token for that).

Reply:
Clarified.